uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/util/RenderContextUtilities.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.uberfire.ext.wires.core.grids.client.util;
+
+import java.util.List;
+import java.util.Optional;
+
+import com.ait.lienzo.client.core.shape.Group;
+import com.ait.lienzo.client.core.types.Point2D;
+import org.uberfire.ext.wires.core.grids.client.model.GridColumn;
+import org.uberfire.ext.wires.core.grids.client.widget.context.GridBodyCellEditContext;
+import org.uberfire.ext.wires.core.grids.client.widget.grid.GridWidget;
+import org.uberfire.ext.wires.core.grids.client.widget.grid.renderers.grids.GridRenderer;
+import org.uberfire.ext.wires.core.grids.client.widget.grid.renderers.grids.impl.BaseGridRendererHelper;
+
+public class RenderContextUtilities {
+
+    public static GridBodyCellEditContext makeRenderContext(final GridWidget gridWidget,
```

Comment:
Just a thought - and sorry, I was pondering this while building the PRs locally for testing - since you've removed the overloaded method that does not need a `rp` is it worth adding some JavaDocs saying `rp` is optional; or reinstate the overloaded method (and use that instead of `null` in the usages in the other PRs)?

Reply:
done, here and in kie-wb-common part, no changes needed in drools-wb part